holoviews/plotting/mpl/renderer.py

```diff
@@ -82,38 +82,12 @@ class MPLRenderer(Renderer):
     mode = param.ObjectSelector(default='default', objects=['default'])
 
 
-    mode_formats = {'fig':     {'default': ['png', 'svg', 'pdf', 'html', None, 'auto']},
-                    'holomap': {'default': ['widgets', 'scrubber', 'webm','mp4', 'gif',
-                                            'html', None, 'auto']}}
```

Comment:
Have you checked the output magic still works? I suspect the old format might have been expected there...

Reply:
Yeah I made the change there too.